src/outputs/epic-games/client.ts

```diff
@@ -1,38 +1,30 @@
 import axios from "axios";
-import { EpicGamesMapperHelper } from "../../helpers/mappers/epic-games.mapper";
-import { ClientInterface, ElementToSendInterface, EpicGamesDatasInterface } from "../../interfaces/client.interface";
+import { ClientInterface, EpicGamesDatasInterface } from "../../interfaces/client.interface";
+import { ClientService } from "../../services/client.service";
 import clientInformations from "./../../../data/client.json";
-import fs from "fs";
 
-export default class ClientEpicGames {
-    private static clientInformations = clientInformations.epicGames;
-    private static clientInterface: ClientInterface = {
-        url: `${this.clientInformations.baseUrl}/${this.clientInformations.endpoint}`,
-        params: this.clientInformations.params
-    };
-    private static data: EpicGamesDatasInterface;
+export class ClientEpicGames {
```

Comment:
I think that it should be something like:
```suggestion
export class EpicGamesClient {
```
... if we want to follow the `SomethingInterface` or `GamesClient` rules.

Reply:
Done